packages/features/bookings/components/event-meta/Locations.tsx

```diff
@@ -21,9 +21,10 @@ export const EventLocations = ({ event }: { event: PublicEvent }) => {
   };
   const eventLocationType = getEventLocationType(locations[0].type);
   const icon = (locations.length > 1 || !eventLocationType?.iconUrl) ? MapPin : eventLocationType.iconUrl;
+  const isDark = !eventLocationType?.iconUrl?.startsWith("/app-store") ? true : false;
```

Comment:
you dont need ? true : false for boolean statements

Reply:
yes yes sorry my mistake. Let me fix it.